huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_account_test.go

```diff
@@ -0,0 +1,263 @@
+		},
+	})
+}
+
+func testAccCheckRdsAccountDestroy(s *terraform.State) error {
+	c := acceptance.TestAccProvider.Meta().(*config.Config)
+	client, err := c.HcRdsV3Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return fmt.Errorf("Error creating RDS client: %s", err)
+	}
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "huaweicloud_rds_account" {
+			continue
+		}
+
+		// Split instance_id and user from resource id
+		parts := strings.SplitN(rs.Primary.ID, "/", 2)
+		if len(parts) != 2 {
+			return fmt.Errorf("invalid id format, must be <instance_id>/<user>")
+		}
+		instance_id := parts[0]
+		user_name := parts[1]
+		// items on every page, [1, 100]
+		limit := int32(100)
+		// List all db users
+		request := &model.ListDbUsersRequest{
+			InstanceId: instance_id,
+			Limit:      limit,
+			Page:       int32(1),
+		}
+
+		for {
+			response, err := client.ListDbUsers(request)
+			if err != nil {
+				return nil
+			}
+			users := *response.Users
+			if len(users) == 0 {
+				break
+			} else {
+				request.Page += 1
+				for _, user := range users {
+					if user.Name == user_name {
+						return fmt.Errorf("Rds account still exists")
+					}
+				}
+			}
+		}
+	}
+
+	return nil
+}
+
+func testAccCheckRdsAccountExists(n string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("no ID is set")
+		}
+
+		c := acceptance.TestAccProvider.Meta().(*config.Config)
+		client, err := c.HcRdsV3Client(acceptance.HW_REGION_NAME)
+		if err != nil {
+			return fmt.Errorf("error creating RDS client: %s", err)
+		}
+
+		// Split instance_id and user from resource id
+		parts := strings.SplitN(rs.Primary.ID, "/", 2)
+		if len(parts) != 2 {
+			return fmt.Errorf("invalid id format, must be <instance_id>/<user>")
+		}
+		instance_id := parts[0]
+		user_name := parts[1]
+		// items on every page, [1, 100]
+		limit := int32(100)
+		// List all db users
+		request := &model.ListDbUsersRequest{
+			InstanceId: instance_id,
+			Limit:      limit,
+			Page:       int32(1),
+		}
+
+		for {
+			response, err := client.ListDbUsers(request)
+			if err != nil {
+				return fmt.Errorf("error listing RDS db users: %s", err)
+			}
+			users := *response.Users
+			if len(users) == 0 {
+				break
+			} else {
+				request.Page += 1
+				for _, user := range users {
+					if user.Name == user_name {
+						return nil
+					}
+				}
+			}
+		}
+
+		return fmt.Errorf("rds account not found")
+	}
+}
+
+func testRdsAccount_basic(rName string) string {
+	return fmt.Sprintf(`
+data "huaweicloud_availability_zones" "test" {}
+
+resource "huaweicloud_vpc" "test" {
+  name = "%[1]s"
+  cidr = "192.168.0.0/16"
+}
+
+resource "huaweicloud_vpc_subnet" "test" {
+  name          = "%[1]s"
+  cidr          = "192.168.0.0/24"
+  gateway_ip    = "192.168.0.1"
+  vpc_id        = huaweicloud_vpc.test.id
```

Comment:
The problems of indentation can be solved with the help of tools.

Reply:
done